<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDate</set>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDate</set>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="jobDesc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="PONum">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseCompany">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="PONeeded">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>PO Needed?</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createInvoice">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Invoice</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseUser">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="addUser"/>
    <addaction name="addCompany"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editUser"/>
    <addaction name="editCompany"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="addUser">
   <property name="text">
    <string>New User</string>
   </property>
   <property name="statusTip">
    <string>Add a new user</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="addCompany">
   <property name="text">
    <string>New Company</string>
   </property>
   <property name="statusTip">
    <string>Add a new company</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Invoice">
   <property name="text">
    <string>New Invoice</string>
   </property>
   <property name="statusTip">
    <string>Create a new invoice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="editCompany">
   <property name="text">
    <string>Company</string>
   </property>
  </action>
  <action name="editUser">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PONeeded</sender>
   <signal>clicked(bool)</signal>
   <receiver>createInvoice</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseUser</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>createInvoice</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>664</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseCompany</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>createInvoice</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>659</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
